app/src/modules/settings/routes/roles/add-new.vue

```diff
@@ -65,16 +50,6 @@ function useSave() {
 					<div class="field full">
 						<v-input v-model="roleName" autofocus :placeholder="t('role_name') + '...'" @keyup.enter="save" />
```

Comment:
The UI does not correctly limit the user input to the maximum length that the backend will accept (100 chars) leading to a cryptic error if attempting  a longer name.
```suggestion
						<v-input v-model="roleName" autofocus :placeholder="t('role_name') + '...'" @keyup.enter="save" maxLength="100" />
```
![firefox_UO7HVFi9A2](https://github.com/directus/directus/assets/9389634/be1210ec-cb0b-48bf-b37e-8cc4bbb963da)